### PR TITLE
Update preferred ciphers

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

File: pkg/cmd/server/crypto/crypto.go [hunk 1]
```diff
@@ -42,12 +42,38 @@ func SecureTLSConfig(config *tls.Config) *tls.Config {
...
+			// TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA
+			// TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+			// TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+			// TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+			// TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+
 			tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
 			tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+			tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
```
Comment (liggitt) [inline pkg/cmd/server/crypto/crypto.go]: we missed these in the original PR, not sure how

File: pkg/cmd/server/crypto/crypto.go [hunk 2]
```diff
@@ -42,12 +42,38 @@ func SecureTLSConfig(config *tls.Config) *tls.Config {
...
 			tls.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
 			tls.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
 			tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
 			tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
+			tls.TLS_RSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
```
Comment (liggitt) [inline pkg/cmd/server/crypto/crypto.go]: these were new in go 1.6:
```
$ diff ~/.gvm/gos/go1.5.3/src/crypto/tls/cipher_suites.go ~/.gvm/gos/go1.6.2/src/crypto/tls/cipher_suites.go 
268a271,272
> 	TLS_RSA_WITH_AES_128_GCM_SHA256         uint16 = 0x009c
> 	TLS_RSA_WITH_AES_256_GCM_SHA384         uint16 = 0x009d
```

**Comment by liggitt** [comment]
[test]

**Comment by liggitt** [comment]
@smarterclayton post-1.3

**Comment by liggitt** [comment]
flake #10002 [test]

**Comment by liggitt** [comment]
@smarterclayton bump

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**